packages/valory/skills/task_execution/behaviours.py

```diff
@@ -240,6 +240,7 @@ def _check_for_new_reqs(self) -> None:
         if from_block is None:
             # set the initial from block
             self._populate_from_block()
+            self.params.req_type = "legacy"
```

Comment:
Parameters should be frozen. Is this an exception in this specific service?
In any case, this should be an `Enum` to avoid human error.